io/transfer/transfer.go

```diff
@@ -107,3 +110,39 @@ type DataEvent struct {
 }
 
 func (DataEvent) ImplementsEvent() {}
+
+// SchemeOp registers a tag as a lister for a scheme updates.
+// Use multiple SchemeOps if a tag supports multiple schemes,
+// or use an empty scheme to listen for all schemes.
+//
+// The app must be compiled with the `-schemes <scheme>` flag,
+// in gogio, to be able to receive URLEvents.
```

Comment:
Nit:

// Note that only schemes registered with the platform receive URLEvents. Use a packaging tool
// such as [gioui.org/cmd/gogio] to register schemes.